ui/pages/Project/selectors.js

```diff
@@ -82,28 +95,34 @@ export const getProjectAnalysisGroupIndividualsByGuid = createSelector(
     }), {}),
 )
 
-export const getProjectAnalysisGroupSamplesByGuid = createSelector(
-  getSamplesByGuid,
-  getProjectAnalysisGroupIndividualsByGuid,
-  (samplesByGuid, individualsByGuid) =>
-    Object.values(individualsByGuid).reduce((acc, individual) => ({
-      ...acc,
-      ...individual.sampleGuids.reduce((sampleAcc, sampleGuid) => (
-        { ...sampleAcc, [sampleGuid]: samplesByGuid[sampleGuid] }
-      ), {}),
-    }), {}),
+export const getProjectAnalysisGroupSamplesByTypes = createSelector(
+  getProjectSamplesByGuid,
+  getSamplesByFamily,
+  getCurrentAnalysisGroup,
+  (projectSamplesByGuid, samplesByFamily, analysisGroup) =>
+    (analysisGroup ? analysisGroup.familyGuids.reduce(
+      (acc, familyGuid) => ([...acc, ...(samplesByFamily[familyGuid] || [])]), [],
+    ) : Object.values(projectSamplesByGuid)).reduce((acc, sample) => {
+      const loadedDate = (sample.loadedDate).split('T')[0]
+      const typeKey = `${sample.sampleType}__${sample.datasetType}`
+      if (!acc[typeKey]) {
+        acc[typeKey] = {}
+      }
+      acc[typeKey][loadedDate] = (acc[typeKey][loadedDate] || 0) + 1
+      return acc
+    }, {}),
 )
 
 export const getProjectAnalysisGroupMmeSubmissions = createSelector(
   getMmeSubmissionsByGuid,
   getProjectAnalysisGroupFamiliesByGuid,
-  getProjectAnalysisGroupIndividualsByGuid,
+  getIndividualsByFamily,
```

Comment:
`getProjectAnalysisGroupIndividualsByGuid` did not otherwise have to run for the main page render but `getIndividualsByFamily` so refactoring this allows us to skip running that selector and its computation entirely